express-app/views/index.hbs

```diff
@@ -3,7 +3,7 @@
   <head>
     {{> head}}
   </head>
-  <body>  
+  <body class="login-bg">  
```

Comment:
If you see in the template they use it that way. Look in _login.html_  line 18. @erikiado